infrastructure/router/router.go

```diff
@@ -0,0 +1,32 @@
+package router
+
+import (
+	"log"
+	"net/http"
+
+	"github.com/ErickRodriguezWize/academy-go-q42021/interface/controller"
+	"github.com/ErickRodriguezWize/academy-go-q42021/service"
+
+	"github.com/gorilla/mux"
+)
+
+func NewRouter() {
+	r := mux.NewRouter()
+
+	pokemonHandler := controller.NewPokemonController()
+
+	//Pokemon Endpoints
+	r.HandleFunc("/pokemons", pokemonHandler.GetAllPokemons).Methods("GET")
+	r.HandleFunc("/pokemons/{id}", pokemonHandler.GetPokemon).Methods("GET")
+
+	//Server setup
+	srv := &http.Server{
+		Handler: r,
+		Addr: service.GetEnvVariable("IP_ADDRESS") + ":" + service.GetEnvVariable("PORT"),
```

Comment:
You should consider adding a timeout. Just in case some requests might take too much time.